adapters/ddf-adapter/src/main/java/com/connexta/replication/adapters/ddf/DdfNodeAdapter.java

```diff
@@ -182,7 +183,7 @@ public String getSystemName() {
     String systemName;
     if (!results.isEmpty()) {
       systemName =
-          ((MetacardAttribute) ((Map) results.get(0).getRawMetadata()).get("title")).getValue();
+          ((MetadataAttribute) ((Map) results.get(0).getRawMetadata()).get("title")).getValue();
```

Comment:
Why is it that we're switching all of these MetacardAttributes to MetadataAttributes?

Reply:
It's just a rename of the class since it is no longer specific to DDF and was moved to the commons module.